ImperatorToCK3/Source/Mappers/GovernmentMapper/GovernmentMapper.cpp

```diff
@@ -0,0 +1,40 @@
+#include "GovernmentMapper.h"
+#include "Log.h"
+#include "ParserHelpers.h"
+#include "GovernmentMapping.h"
+
+mappers::GovernmentMapper::GovernmentMapper()
+{
+	LOG(LogLevel::Info) << "-> Parsing government mappings.";
+	registerKeys();
+	parseFile("configurables/government_map.txt");
+	clearRegisteredKeywords();
+	LOG(LogLevel::Info) << "<> Loaded " << impToCK3GovernmentMap.size() << " government links.";
+}
+
+mappers::GovernmentMapper::GovernmentMapper(std::istream& theStream)
+{
+	registerKeys();
+	parseStream(theStream);
+	clearRegisteredKeywords();
+}
+
+void mappers::GovernmentMapper::registerKeys()
+{
+	registerKeyword("link", [this](const std::string& unused, std::istream& theStream) {
+		const GovernmentMapping theMapping(theStream);
+		for (const auto& imperatorGovernment: theMapping.impGovernments)
+		{
+			if (theMapping.ck3Government) impToCK3GovernmentMap.emplace(imperatorGovernment, *theMapping.ck3Government);
```

Comment:
why if? If there's no ck3gov, throw a runtime exception and die. Let someone fix it, these are our mappings.